packages/venia-concept/src/components/ToastContainer/__stories__/toasts.js

```diff
@@ -33,7 +33,7 @@ const ToastEmitter = ({
 
     useEffect(() => {
         addToast(toastProps);
-    }, []);
+    }, []); // eslint-disable-line
```

Comment:
You mentioned this would be an exception to the rule; a comment may be good here so the community knows of valid reasons to suppress this lint rule. I think I'm just going to approve though, IMO Stories aren't API, and really shouldn't be used as implementation references.

Reply:
I added a comment here and in the other place where we added an `eslint-disable-line`.